ArduPlane/quadplane.cpp

```diff
@@ -2792,6 +2792,10 @@ int8_t QuadPlane::forward_throttle_pct(void)
         // lidar could cause the aircraft not to be able to
         // approach the landing point when landing below the takeoff point
         vel_forward.last_pct = vel_forward.integrator;
+    } else if (in_vtol_land_descent() && poscontrol.state == QPOS_LAND_FINAL && motors->limit.throttle_lower) {
```

Comment:
The motors don't always stay at the lower throttle limit when landing on the ground depending on the environmental conditions, which would lead to the forward motor spinning up again. Why not in the next section just turn it off  if rangefinder_landing, and we are indicating being out of range low?

Reply:
I've added another patch that covers this special case in height_above_ground(). That should fix several corner cases